testing/unit/test_configurator.py

```diff
@@ -0,0 +1,47 @@
+"""Unit tests for configurator"""
+
+import unittest
+import os
+
+from daq.configurator import Configurator, print_config
+
+def dict_to_str(obj):
```

Comment:
dict_to_conf_str -- it's not just a "str" which could be anything... it's more specific than that!

Reply:
Addressed.